changelogs/fragments/2014-allow-root-for-kibana-plugin.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - add parameter to kibana_plugin for passing --allow-root flag to kibana and kibana-plugin commands
```

Comment:
```suggestion
  - add parameter to kibana_plugin for passing --allow-root flag to kibana and kibana-plugin commands (https://github.com/ansible-collections/community.general/pull/2014).
```

Reply:
This is much better, thanks @felixfontein !

Reply:
Thanks for the quick help and for the upbeatness :)